lib/kintsugi/cli.rb

```diff
@@ -205,5 +202,14 @@ def create_root_command
         description: nil
       )
     end
+
+    def subcommands_descriptions(subcommands)
+      longest_subcommand_name = subcommands.keys.max_by(&:length)
+
+      subcommands.map do |command_name, command|
+        spaces = " " * (3 + longest_subcommand_name.length - command_name.length)
+        "    #{command_name}:#{spaces}#{command.description}"
```

Comment:
```suggestion
      subcommand_length = subcommands.keys.map(&:length).max + 4
      format = "%-#{subcommand_length}s %s"

      subcommands.map do |command_name, command|
        format % ["#{command_name}:", command.description]
```

I think this should work

Reply:
Doesn't work if there are spaces before `command_name`, and more difficult to understand IMO

Reply:
of course it does, I just don't understand why you're reinventing padding

Reply:
right, sorry, long day :)